docker/Dockerfile.make

```
@@ -2,7 +2,14 @@
 FROM golang:1.18-alpine as golang_image
 FROM node:16-alpine
 
-RUN apk add make gcc musl-dev
+RUN apk add --no-cache --update make gcc binutils-gold musl-dev
+
+ENV PNPM_HOME=/root/.local/share/pnpm
+ENV PATH=$PATH:$PNPM_HOME
+
+RUN apk add --no-cache --update curl && \
+    curl -fsSL "https://github.com/pnpm/pnpm/releases/latest/download/pnpm-linuxstatic-x64" -o /bin/pnpm && chmod +x /bin/pnpm && \
+    apk del curl
```

```suggestion
RUN corepack enable
```

As we do it in our pipelines: https://github.com/woodpecker-ci/woodpecker/blob/master/.woodpecker/web.yml#L14-L15